rules/windows/process_creation/proc_creation_win_powershell_cl_loadassembly.yml

```diff
@@ -0,0 +1,26 @@
+title: Assembly Loading Via CL_LoadAssembly.ps1
+id: c57872c7-614f-4d7f-a40d-b78c8df2d30d
+status: experimental
+description: Detects calls to "LoadAssemblyFromPath" or "LoadAssemblyFromNS" that are part of the "CL_LoadAssembly.ps1" script. Which can be abused to load different assemblies and bypass App locker controls.
```

Comment:
```suggestion
description: Detects calls to "LoadAssemblyFromPath" or "LoadAssemblyFromNS" that are part of the "CL_LoadAssembly.ps1" script. This can be abused to load different assemblies and bypass App locker controls.
```